keyboards/gboards/g/keymap_combo.h

```diff
@@ -66,10 +69,35 @@ void process_combo_event(uint16_t combo_index, bool pressed) {
     }
 
     // Allow user overrides per keymap
-#if __has_include("inject.h") 
+#if __has_include("inject.h")
 # include "inject.h"
 #endif
 }
 #undef COMB
 #undef SUBS
 #undef TOGG
+
+// Allow reference layers per layer.
+#define COMB BLANK
+#define SUBS BLANK
+#define TOGG BLANK
+
+#undef COMBO_REF_LAYER
+#define COMBO_REF_LAYER REF_LAYER
+
+uint8_t combo_ref_from_layer(uint8_t layer){
+  switch (get_highest_layer(layer_state)){
```

Comment:
Does this need to be `get_highest_layer(layer_state)` now (when the combo code no longer handles `COMBO_REF_DEFAULT`)?
```suggestion
  switch (layer) {
```

Reply:
yep.  another thing I missed in my confusion of making it work for me, and making it work here. I didnt realize I had two versions.

Reply:
I already had this one..